#! /bin/sh /usr/share/dpatch/dpatch-run
## 38_gnu.dpatch by Samuel Thibault <samuel.thibault@ens-lyon.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: GNU/Hurd support. bz#408745

@DPATCH@

--- mozilla/config/rules.mk	2007-12-30 20:47:00.120197000 +0000
+++ mozilla/config/rules.mk	2007-12-30 20:47:27.127970000 +0000
@@ -506,6 +506,14 @@
 endif 
 
 #
+# GNU doesn't have path length limitation
+#
+
+ifeq ($(OS_ARCH),GNU)
+OS_CPPFLAGS += -DPATH_MAX=1024 -DMAXPATHLEN=1024
+endif
+
+#
 # BeOS: add -Bsymbolic flag for components
 # 
 ifeq ($(OS_ARCH),BeOS)
